src/containers/App/index.js

```diff
@@ -130,11 +130,16 @@ export default class App extends Component {
 const asyncConnectProps = [{
   promise: ({ store: { dispatch, getState } }) => {
     const promises = [];
+    const state = getState();
 
-    if (!isAuthLoaded(getState())) {
+    if (!isAuthLoaded(state)) {
       promises.push(dispatch(loadAuth()));
     }
 
+    if (state.auth.user) {
+      promises.push(dispatch(loadGroups()));
```

Comment:
Why you had to move `loadGroups()` to the `App` container? You're not even passing it as a `prop` to the children. My concern here is that the `Group/Base` container will look for `groups` in the state relying that `App` already loaded them here.

Can we at least pass them to the children here?